src/Elastic.Apm.AspNetCore/WebRequestTransactionCreator.cs

```diff
@@ -187,6 +187,10 @@ internal static void StopTransaction(Transaction transaction, HttpContext contex
 
 						if (!string.IsNullOrWhiteSpace(name)) transaction.Name = $"{context.Request.Method} {name}";
 					}
+					else if (context.Response.StatusCode == StatusCodes.Status404NotFound)
+					{
+						transaction.Name = $"{context.Request.Method} Unknown Route";
```

Comment:
nit: [the spec](https://github.com/elastic/apm/blob/master/specs/agents/tracing-instrumentation-http.md#http-transactions) uses non capital letters, so let's stick to that.

```suggestion
						transaction.Name = $"{context.Request.Method} unknown route";
```